lib/Dialect/TritonGPU/Transforms/AccelerateAMDMatmul.cpp

```diff
@@ -256,19 +342,24 @@ class TritonAMDGPUAccelerateMatmulPass
 public:
   TritonAMDGPUAccelerateMatmulPass() = default;
   TritonAMDGPUAccelerateMatmulPass(int matrixCoreVersion,
-                                   int matrixInstructionSize) {
+                                   int matrixInstructionSize,
+                                   bool supportWMMA) {
```

Comment:
It's better to handle wmma check the same way as mfma check

Reply:
Do you mean provide version of RDNA?